### ISSUE TITLE
accuracy_checker and pot[Bug]

### ISSUE THREAD

**Issue by guojin-yan** [openvinotoolkit/openvino#11571]
##### System information (version)
- OpenVINO => 2022.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2022
- Problem classification: accuracy_checker and pot
- Model name: ResNet50 (flower classification)

My main problem is to refer to the end to End-to-end Command-line Interface Example <https://docs.openvino.ai/latest/pot_configs_examples_README.html> in openvino document to realize the transformation of flower classification model from F32 to int8 format， but there are problems in the step of accuracy detection; Another problem is that after pot conversion, the model can predict, but the result is wrong.

First, the yaml file is configured according to the document requirements. The file contents are as follows：
![image](https://user-images.githubusercontent.com/91110433/165055077-37ba6dc1-1f63-49f1-8d1f-19949935c998.png)

In the command prompt window, enter the following：
![image](https://user-images.githubusercontent.com/91110433/165055370-c05bccc4-3579-4ee5-aba5-ff51d6483c02.png)

But the prediction result is very low：
![image](https://user-images.githubusercontent.com/91110433/165055530-e797481d-0efd-41a8-b57f-c0ed49a9e672.png)

Next, perform the pot conversion，The JSON file is as follows：
![image](https://user-images.githubusercontent.com/91110433/165055868-ab857065-16b8-48c1-819f-8d791b8a3080.png)

In the command prompt window, enter the following：
![image](https://user-images.githubusercontent.com/91110433/165056222-d9646a92-cd8f-4d8a-a3c2-d9532d96233a.png)

But the prediction result is very low：
![image](https://user-images.githubusercontent.com/91110433/165056479-9f5d0ad4-8ec2-4d79-a3a6-e6cfda352010.png)

Later, I read the model through openvino tool and then reasoning. The reasoning result is still wrong, but the reasoning result of the original model is correct in the initial use, the accuracy detected by the accuracy detection tool is very low, and the model detection result after pot is still very low.

**Comment by dlod-openvino** [comment]
Add the datasets: flowers102 and the whole training process as below
https://github.com/PaddlePaddle/PaddleClas/blob/release/static/docs/en/tutorials/quick_start_en.md

**Comment by Iffa-Meah** [comment]
@guojin-yan , could you share your model files and link to your dataset so that we could test it from our end?

**Comment by guojin-yan** [comment]
@Iffa-Meah 
Model files and data sets are placed in Baidu online disk sharing link：
链接：https://pan.baidu.com/s/1ED6AcZezUt_july-3jMiBQ 
提取码：yyyy

**Comment by Iffa-Meah** [comment]
![image](https://user-images.githubusercontent.com/79643476/165462578-22ae9dc5-85fd-42e3-a4c6-09e7e2faaee5.png)

@guojin-yan , it seems that I need extraction code to access the file

**Comment by guojin-yan** [comment]
@Iffa-Meah 
extraction code: yyyy

**Comment by guojin-yan** [comment]
@Iffa-Meah 
The lable file in the data set was forgotten in it. You can find it in this web disk connection:
link：https://pan.baidu.com/s/1Uvm2NuDsXPgMSGbfL4M8WA
extraction code：yyyy
![IMG_20220427_152730](https://user-images.githubusercontent.com/91110433/165465157-0700ace4-a361-4b97-b472-b6ba0051cb50.jpg)

If you can't download it, you can get it from my Github project, which is linked to: github: https://github.com/guojin-yan/ResNet50_INT8_OpenVINO.git
Dataset file and download method are in dataset folder.
![IMG_20220427_153350](https://user-images.githubusercontent.com/91110433/165465938-c4ffd44d-059e-4ea8-b985-6ac3cd702a5e.jpg)
![IMG_20220427_153406](https://user-images.githubusercontent.com/91110433/165465957-d062b77e-39ae-4228-a545-ea102529f8c4.jpg)

**Comment by guojin-yan** [comment]
@Iffa-Meah 
If you can't download it, you can get it from my Github project, which is linked to: github: https://github.com/guojin-yan/ResNet50_INT8_OpenVINO.git
Dataset file and download method are in dataset folder.
Model file is in modelfolder.
![image](https://user-images.githubusercontent.com/91110433/165472523-463daa72-93f1-4571-9934-426124712e48.png)

**Comment by Iffa-Meah** [comment]
@guojin-yan ,

I tested your model (both FP32 and INT8).

POT:
![pot_LI](https://user-images.githubusercontent.com/79643476/165696280-944ee801-8aad-42f8-a676-90d0eb7646e2.jpg)



FP32 accuracy:
<img width="612" alt="accuracy" src="https://user-images.githubusercontent.com/79643476/165695902-7ff2c5d1-9a65-44d3-b629-236e53cf1ec0.png">

INT8 accuracy:

<img width="616" alt="accuracyINT8" src="https://user-images.githubusercontent.com/79643476/165696026-e767c4a7-b45f-4c32-85a4-28eb31a8edb7.png">

FP32 performance (benchmark_app):
<img width="620" alt="benchmark" src="https://user-images.githubusercontent.com/79643476/165696096-d69c0ffe-1070-4aa1-a3bd-48132de3f88f.png">


INT8 performance (benchmark_app):
<img width="612" alt="benchmarkINT8" src="https://user-images.githubusercontent.com/79643476/165696179-8780bc31-8541-4e2a-ae67-2c0692710323.png">


Performance-wise, the INT8 does seem to do its part since I could see the latency is halved compared to the FP32.
However, bear in mind that once the model is converted into INT8, the original size of the model is also reduced and the trade-off for this is the accuracy.

In short, these depends on your use case:

1. If you are working on some sort of Healthcare related project, you'll definitely need to have accurate results. Hence, you need to use the full precision model FP32 and bear with the slower inferencing speed.
2. If real-time data is involved, such as smart parking, where you need to process data faster regardless the accuracy, then optimizing/quantizing (INT8) would be useful

**Comment by brmarkus** [comment]
Did you do the pot-quantization with "Default Quantization" only?
Can you try to use "accuracy-aware quantization" as well? You can specify the max. acceptable drop in accuracy. The quantization might take much longer as it might require more optimizations - and maybe even not quantizing certain layers but keeping them in the original precision if it causes too much of accuracy-drop.
